en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -0,0 +1,183 @@
+2.03 Changelog
+^^^^^^^^^^^^^^
+
+Changes / additions to IATI Standard for version 2.03
+
+Note: Version 2.03 was released on Monday 19th February 2018.
+
+.. contents::
+
+
+.. _2_03_schema_changes:
+
+Schema Changes
+==============
+
+A full diff of all schema changes in 2.03 can be found in the `IATI GitHub <https://github.com/IATI>`__
+
+- `Activities Schema <https://github.com/andylolz/IATI-Schemas/pull/1/commits/97732bdaa5543aa657d2a667f5b23fdfd9ac8f36>`__
```

Comment:
Should only link to things on Secretariat-owned repos.

Reply:
Previous Changelogs use ` https://github.com/IATI/IATI-Schemas/compare/version-2.02...version-2.03#files_bucket` format

Reply:
fd78daf8c59b311e24964edbcd2488331b8e60fd